### ISSUE TITLE
Running on new iobroker?

### ISSUE THREAD

**Issue by hugocrank** [AttackStrawbery/iobroker.sma-speedwire#17]
there is no chance to use this adapter under newest iobroker? i get only "Timeout" :-( SMA-EM adapter is working.
--> fresh installation of iobroker and ubuntu.

**Comment by bobmorane77** [comment]
i think this is related to issue #16 
you have js-controller 5.x?

**Comment by hugocrank** [comment]
yes, i don´t know how to install older version oder downgrade... --> correct me i found the way to install older version for testing.

**Comment by bobmorane77** [comment]
i took another was, i use docker for som projects, so i installed a iobroker docker image with js controller 4.x and just the sma adapter, then writing in my existing influx - done
yes i know 2 iobroker instances, but my raspi has enough power to handle that

**Comment by hugocrank** [comment]
can you send me the link for the dockerimage? i have try to install with js.4 and speedwireadapter but it show the same error : Timeout.

**Comment by bobmorane77** [comment]
i use buanet/iobroker:v7.2.0 in hub.docker.com
version 8.x is not working

**Comment by hugocrank** [comment]
thank you, have testet by my configuration and installed the same Version in Docker. Not successful seems to be a problem with the sma Inverter to.

**Comment by rg-engineering** [comment]
under https://github.com/rg-engineering/ioBroker.sma-speedwire there is a version running with latest ioB... but still a lot of to do's...
